app/models/user.rb

```diff
@@ -46,7 +46,7 @@ def channels
   end
 
   def active_request
-    received_requests.reorder(created_at: :desc).first
+    received_requests.reorder(created_at: :desc).first || Request.reorder(created_at: :desc).first
```

Comment:
I think that’s just equivalent to the following:

```suggestion
    Request.reorder(created_at: :desc).first
```

Two cases:

* A user has never received a request, then it’ll be the most recent request as a fallback
* A user has received a request, then it’ll be the most recent request, as all user’s receive the same requests.

Do we still need an active request per user? I think from a UX perspective, right now, editors already expect the question to be the same for all users at the same time. So why don't we use `Request.active` as we did previously?

Reply:
@drjakob @FrauCsu I talked with @tillprochaska about this problem. I don't really like this solution in this PR nor do I like the solution proposed here by Till.

For the future: Would be a reasonable solution to automatically send the user on registration (via Mail or Telegram) the most recent request as a message? So the user always knows, what he/she is responding to?

I'm going to merge this as it is for now.